pkg/services/searchV2/object/testdata/dash_raw.jsonc

```diff
@@ -276,9 +276,9 @@
             61778,
             10392,
             25047,
-            28885,
+            28884,
```

Comment:
@george124816 looks like there's still some changes in here that you didn't want to commit